scipy/interpolate/tests/test_bsplines.py

```diff
@@ -324,6 +343,13 @@ def test_integral(self):
         assert_allclose(b.integrate(-1, 1), 0)
         assert_allclose(b.integrate(-1, 1, extrapolate=True), 0)
         assert_allclose(b.integrate(-1, 1, extrapolate=False), 0.5)
+        assert_allclose(b.integrate(-1, 1, extrapolate='periodic'), 1)
+        assert_allclose(b.integrate(1, -1, extrapolate='periodic'), -1)
```

Comment:
I think these two lines are enough to test the swapping of `a` and `b`. What do you say @nmayorov ?